easybuild/easyblocks/c/clang.py

```diff
@@ -84,9 +85,11 @@ def extra_options():
             'skip_all_tests': [False, "Skip running of tests", CUSTOM],
             # The sanitizer tests often fail on HPC systems due to the 'weird' environment.
             'skip_sanitizer_tests': [True, "Do not run the sanitizer tests", CUSTOM],
-        }
-
-        return CMakeMake.extra_options(extra_vars)
+        })
+        extra_vars['build_type'][0] = 'Release'
```

Comment:
I did not want to change existing behavior, hence I set it here. I think it doesn't really matter, because this is the stage0 compiler used to build the real compiler later on, so using Release here makes sense. As far as I understood the process (stage0 compiler used to build final compiler) it shouldn't affect the final binary.

But now that you mention it: We need to handle the build_type in line 336 as it was added to configopts before and that is used later here.

So: Don't overwrite the default here, use Release for the stage0 (line 229) and Release/Debug in line 336 for stage1

Reply:
Why would using `Debug` for the stage0 compiler be problematic? I don't think it would be?

Reply:
Not problematic, just unnecessarily slower for no gain.